src/test/java/com/conveyal/gtfs/GTFSTest.java

```diff
@@ -428,21 +426,12 @@ private boolean runIntegrationTestOnZipFile(
 
         // Verify that making a snapshot from an existing feed database, then exporting that snapshot to a GTFS zip file
         // works as expected
-        try {
-            LOG.info("copy GTFS from created namespace");
-            SnapshotResult copyResult = GTFS.makeSnapshot(namespace, dataSource);
-            assertThatSnapshotIsErrorFree(copyResult);
-            LOG.info("export GTFS from copied namespace");
-            File tempFile = exportGtfs(copyResult.uniqueIdentifier, dataSource, true);
-            assertThatExportedGtfsMeetsExpectations(tempFile, persistenceExpectations, true);
-        } catch (IOException e) {
-            e.printStackTrace();
-            TestUtils.dropDB(testDBName);
-            return false;
-        } catch (AssertionError e) {
-            TestUtils.dropDB(testDBName);
-            throw e;
-        }
+        boolean snapshotIsOk = assertThatSnapshotIsSuccessful(namespace, dataSource, testDBName, persistenceExpectations, false);
```

Comment:
For this test and the one just below, I kinda want to write `if (!assert...()) return false`.